docs/sdk/examples/assets/pastefromoffice/pastefromoffice.js

```diff
@@ -0,0 +1,162 @@
+/**
```

Comment:
Note that we are just using this file directly on the page, so every variable or function is available from the global scope if not hidden. E.g. see
![image](https://user-images.githubusercontent.com/10279586/130437712-70492215-91bc-4ea2-9df8-2ac8b0ceede9.png)
To avoid polluting the global namespace (window) we should use IIFE (Immediately Invoked Function Expression) pattern here, e.g.:
```js
( function() {
  // Private variables and functions:
  var sources = [ ... ]
  function x() { ... }
  function y() { ... }
  // Expose public API:
  window.createNotification = function () { ... }
} )();
```